tools/opt/PassBuilder.h

```diff
@@ -0,0 +1,314 @@
+//===- Parsing, selection, and construction of pass pipelines --*- C++ -*--===//
```

Comment:
We don't need this file for same reason as described in `tools/opt/PassBuilder.cpp`.